src/openforms/submissions/templates/admin/submissions/submission/logs.html

```diff
@@ -17,37 +17,61 @@
 {% endblock %}
 
 {% block content %}
-    <h1>{% trans  "Logic logs" %}</h1>
     {% block field_sets %}
         {% if not logs_activity %}
             {% trans "No logic rules for that submission" %}
         
         {% else %}
             {% for log in logs_activity  %}
-                <span class="timestamp-log"></span>
                 <fieldset class="module aligned collapse collapsed fieldset-log">
                     <h2>
                         <time datetime="{{ log.timestamp.isoformat }}" title="{{ log.timestamp|date:'Y-m-d H:i:s' }}">
                             {% blocktrans with since=log.timestamp|timesince %}{{ since }} ago{% endblocktrans %}
                         </time>
                         - {% trans 'Logic' %} -
                     </h2>
+                    <h4> {% trans "Variables" %} </h4>
+                    <table class="logic-logs logic-logs--variables">
+                        <thead>
+                            <th class="logic-logs__col-header logic-logs__col-header--name"> {% trans "Property name" %} </th>
+                            <th class="logic-logs__col-header logic-logs__col-header--value"> {% trans "Value" %} </th>
+                        </thead>
+                        <tbody>
+                            {% for key, value in log.extra_data.input_data.items  %}
+                            <tr>
+                                <td colspan="1">{{ value.stepName }}: <code>{{ key }}</code> &#40;{{ value.label }}&#41;</td>
```

Comment:
`colspan="1"` is the default - you should only specify it if you span multiple columns.